### DISCUSSION TITLE
Favourites functionality does not work properly

### DISCUSSION THREAD

**Discussion by SarahJoyceMorgen** [neuefische/questions#805] [Web] [answered]
Can you please help us troubleshoot the issue? The toggling functionality works; however, on the favourite page, we are not seeing any changes, and the favorite card component is not being displayed. Additionally, we are encountering an unhandled runtime error with the following details:

```
Source
components/Card/index.js (56:37) @ includes

  54 | ideas,
  55 | }) {
> 56 | const isFavourite = favouriteIdeas.includes(id);
     |                                   ^
  57 | 
  58 | return (
  59 |   <Article>Unhandled Runtime Error
TypeError: Cannot read properties of undefined (reading 'includes')

```

Current State:
[https://github.com/kevinldg/upcycling-app/tree/feature/US9](url)

**Comment by kevinldg** [comment, marked as answer]
Fixed link to the current state: https://github.com/kevinldg/upcycling-app/tree/feature/US9

**Reply by marcbendixen** [reply]
Nice, so in order to show the ideas on the favorite page, it looks like you had to pass the "favoriteIdeas" prop 👍 therefore I mark the questions as solved.
